sympy/logic/tests/test_boolalg.py

```diff
@@ -227,6 +227,13 @@ def test_Equivalent():
     assert Equivalent(Equality(A, B), Equality(B, A)) is true
 
 
+def test_Exclusive():
+    assert Exclusive(False, False, False)
+    assert Exclusive(True, False, False)
+    assert not Exclusive(True, True, False)
+    assert not Exclusive(True, True, True)
```

Comment:
It's better to use e.g. `is True` and `is False` in tests. Most objects `x` in Python will pass an `assert x`.